core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -81,9 +81,9 @@ <h1 class="oppia-exploration-h1"><span class="protractor-test-exploration-header
 {% endblock %}
 
 {% block footer %}
-  {% if not iframed %}
-    <exploration-footer twitter-text="{{DEFAULT_TWITTER_SHARE_MESSAGE_PLAYER}}"></exploration-footer>
-  {% endif %}
+
+    <exploration-footer ng-if="!iframed" twitter-text="{{DEFAULT_TWITTER_SHARE_MESSAGE_PLAYER}}"></exploration-footer>
```

Comment:
`iframed` is defined in Base.js on line number 26. Since, exploration_player/exploration_player.html is extended from base.html, so I guess, it is defined there.

Reply:
That's great! :) 
This part looks good to me!